src/components/tooltip/TooltipDetails.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+
+type DetailProps = {|
+  +label: string,
+  // Only one child is accepted.
+  +children?: void | null | boolean | string | number | React.Element<any>,
+|};
+
+export function TooltipDetail({ label, children }: DetailProps) {
```

Comment:
Please add a code comment here, describing how this component is supposed to be used, especially pointing out the grid layout.

Reply:
will do!

Reply:
```js
/*
 * This component enforces the HTML structure we use in the marker tooltips.
 * This is made necessary by our use of a grid layout that needs this strict
 * structure.
 */
```